Source/Core/Common/StringUtil.cpp

```diff
@@ -449,6 +449,12 @@ void StringPopBackIf(std::string* s, char c)
     s->pop_back();
 }
 
+size_t StringUTF8Size(const std::string& str)
```

Comment:
I think this name could be made a bit clearer, for instance `UTF8CodePointCount`.